.github/workflows/build-test.yml

```diff
@@ -46,55 +47,3 @@ jobs:
                 PARAMETERS_SECRET: ${{ secrets.PARAMETERS_SECRET }}
                 CLOUD_PROVIDER: ${{ matrix.cloud }}
               run: ./ci/test.sh
-    build-test-mac:
```

Comment:
Do we have a TODO to re-enable mac and windows tests as well, or are these intended to be disabled?

Reply:
We only run our code in production on linux machines, and I don't see that changing any time soon. We run locally on Macs, but even that may be going away with dockerization. If we start hitting issues running on Macs, we can put in the work to enable those tests (we'll need to set up some additional testing secrets).